examples/step-64/step-64.cu

```diff
@@ -200,9 +216,14 @@ namespace Step64
   }
 
 
+  // @sect3{Class <code>HelmholtzOperator</code>}
 
-  // The HelmholtzOperator class acts as wrapper for LocalHelmholtzOperator
-  // defining an interface that can be used with linear solvers like SolverCG.
+  // The `HelmholtzOperator` class acts as wrapper for
+  // `LocalHelmholtzOperator` defining an interface that can be used
+  // with linear solvers like SolverCG. In particular, like every
+  // class that implements the interface of a linear operator, it
+  // needs to have a `vmult()` function that performs the product of
+  // the linear operator and a source vector.
```

Comment:
This is more the action or evaluation of the linear operator for a source vector than a product, isn't it?